vscode/Dockerfile-runtime-pbj-python39

```diff
@@ -0,0 +1,81 @@
+#  made up of hundreds of individual components, each of which was 
+#  individually copyrighted.  Each Cloudera open source product is a 
+#  collective work under U.S. Copyright Law. Your license to use the 
+#  collective work is as provided in your written agreement with  
+#  Cloudera.  Used apart from the collective work, this file is 
+#  licensed for your use pursuant to the open source license 
+#  identified above.
+#
+#  This code is provided to you pursuant a written agreement with
+#  (i) Cloudera, Inc. or (ii) a third-party authorized to distribute 
+#  this code. If you do not have a written agreement with Cloudera nor 
+#  with an authorized and properly licensed third party, you do not 
+#  have any rights to access nor to use this code.
+#
+#  Absent a written agreement with Cloudera, Inc. (“Cloudera”) to the
+#  contrary, A) CLOUDERA PROVIDES THIS CODE TO YOU WITHOUT WARRANTIES OF ANY 
+#  KIND; (B) CLOUDERA DISCLAIMS ANY AND ALL EXPRESS AND IMPLIED 
+#  WARRANTIES WITH RESPECT TO THIS CODE, INCLUDING BUT NOT LIMITED TO 
+#  IMPLIED WARRANTIES OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY AND 
+#  FITNESS FOR A PARTICULAR PURPOSE; (C) CLOUDERA IS NOT LIABLE TO YOU, 
+#  AND WILL NOT DEFEND, INDEMNIFY, NOR HOLD YOU HARMLESS FOR ANY CLAIMS 
+#  ARISING FROM OR RELATED TO THE CODE; AND (D)WITH RESPECT TO YOUR EXERCISE 
+#  OF ANY RIGHTS GRANTED TO YOU FOR THE CODE, CLOUDERA IS NOT LIABLE FOR ANY
+#  DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, PUNITIVE OR
+#  CONSEQUENTIAL DAMAGES INCLUDING, BUT NOT LIMITED TO, DAMAGES 
+#  RELATED TO LOST REVENUE, LOST PROFITS, LOSS OF INCOME, LOSS OF 
+#  BUSINESS ADVANTAGE OR UNAVAILABILITY, OR LOSS OR CORRUPTION OF
+#  DATA.
+#
+#
+############################################################################
+
+
+# Start with a Base CML Runtime & upgrade
+# Note: This particular image will use the Python base image, not R as expected. There is a
+# complication with CML that prevents editors and applications from launching if the CML 
+# version of python isn't present. This should be fixed in a future release.
+FROM docker.repository.cloudera.com/cloudera/cdsw/ml-runtime-pbj-workbench-python3.10-standard:2023.05.2-b7
+
+RUN apt update && apt upgrade -y && apt clean && rm -rf /var/lib/apt/lists/*
+
+# Install code-server
+# https://github.com/coder/code-server
+# Install latest version. See https://github.com/coder/code-server/blob/main/install.sh for details
+RUN curl -fsSL https://code-server.dev/install.sh | sh -s --  --version 4.16.1
+
+
+# Create launch script and symlink to the default editor launcher 
+RUN printf "#!/bin/bash\n/usr/bin/code-server --auth=none --bind-addr=127.0.0.1:8090 --disable-telemetry" > /usr/local/bin/vscode
+
+RUN chmod +x /usr/local/bin/vscode
+RUN ln -s /usr/local/bin/vscode /usr/local/bin/ml-runtime-editor
+
+
+# Override Runtime label and environment variables metadata
+ENV ML_RUNTIME_EDITOR="VsCode" \
+    ML_RUNTIME_EDITION="v4.16.1" \
```

Comment:
I wouldn't put a version number in the edition field, this will mess up the Runtime selector UX when you have multiple versions. You can just say "Community" in the edition field.